dockstore-webservice/src/main/java/io/dockstore/webservice/resources/TokenResource.java

```diff
@@ -142,11 +138,23 @@ public TokenResource(TokenDAO tokenDAO, UserDAO enduserDAO, HttpClient client, C
         this.cachingAuthenticator = cachingAuthenticator;
     }
 
+    public static Token createGoogleToken(String accessToken, String refreshToken, long userID, String googleLoginName) {
```

Comment:
~Seems like this can be private~

That said, this looks like just a constructor in disguise. 
Put it in Token and then it would be easier to find and re-use.